### ISSUE TITLE
Migration: More broken links

### ISSUE THREAD

**Issue by guimachiavelli** [meilisearch/documentation#2341]
@bidoubiwa updated her script and found (quite) a few more links.

```json
{
  "https://meilisearch-staging.vercel.app/docs": [
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/what_is_meilisearch/overview": [
    "https://github.com/meilisearch/documentation/edit/main/learn/what_is_meilisearch/overview.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/overview": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/overview.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/indexes": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/indexes.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/documents": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/documents.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/search": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/search.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/multi_search": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/multi_search.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/tasks": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/tasks.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/keys": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/keys.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/settings": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/settings.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/stats": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/stats.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/health": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/health.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/version": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/version.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/api/dump": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/dump.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/errors/overview": [
    "https://github.com/meilisearch/documentation/edit/main/reference/errors/overview.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/reference/errors/error_codes": [
    "https://github.com/meilisearch/documentation/edit/main/reference/errors/error_codes.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/faq": [
    "mailto:privacy@meilisearch.com",
    "https://github.com/meilisearch/documentation/edit/main/resources/faq.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/what_is_meilisearch/features": [
    "https://github.com/meilisearch/documentation/edit/main/learn/what_is_meilisearch/features.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/what_is_meilisearch/philosophy": [
    "https://github.com/meilisearch/documentation/edit/main/learn/what_is_meilisearch/philosophy.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/what_is_meilisearch/sdks": [
    "https://github.com/meilisearch/documentation/edit/main/learn/what_is_meilisearch/sdks.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/what_is_meilisearch/comparison_to_alternatives": [
    "https://opensource.org/node/1099",
    "https://github.com/meilisearch/documentation/edit/main/learn/what_is_meilisearch/comparison_to_alternatives.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/what_is_meilisearch/telemetry": [
    "mailto:privacy@meilisearch.com",
    "https://github.com/meilisearch/documentation/edit/main/learn/what_is_meilisearch/telemetry.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/what_is_meilisearch/search_preview": [
    "https://github.com/meilisearch/documentation/edit/main/learn/what_is_meilisearch/search_preview.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/what_is_meilisearch/language": [
    "https://github.com/meilisearch/charabia/blob/main/CONTRIBUTING",
    "https://github.com/meilisearch/documentation/edit/main/learn/what_is_meilisearch/language.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/getting_started/installation": [
    "https://github.com/meilisearch/documentation/edit/main/learn/getting_started/installation.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/getting_started/quick_start": [
    "https://github.com/meilisearch/documentation/edit/main/learn/getting_started/quick_start.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/meilisearch_101/filtering_and_sorting": [
    "https://github.com/meilisearch/documentation/edit/main/learn/meilisearch_101/filtering_and_sorting.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/meilisearch_101/customizing_relevancy": [
    "https://github.com/meilisearch/documentation/edit/main/learn/meilisearch_101/customizing_relevancy.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/meilisearch_101/getting_ready_for_production": [
    "https://github.com/meilisearch/documentation/edit/main/learn/meilisearch_101/getting_ready_for_production.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/core_concepts/documents": [
    "https://github.com/meilisearch/documentation/edit/main/learn/core_concepts/documents.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/core_concepts/indexes": [
    "https://imdb.com/",
    "https://github.com/meilisearch/documentation/edit/main/learn/core_concepts/indexes.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/core_concepts/relevancy": [
    "https://github.com/meilisearch/documentation/edit/main/learn/core_concepts/relevancy.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/core_concepts/primary_key": [
    "https://github.com/meilisearch/documentation/edit/main/learn/core_concepts/primary_key.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/configuration/instance_options": [
    "https://linuxize.com/post/how-to-set-and-list-environment-variables-in-linux/",
    "https://github.com/meilisearch/documentation/edit/main/learn/configuration/instance_options.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/index_settings/settings": [
    "https://github.com/meilisearch/documentation/edit/main/learn/index_settings/overview.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/index_settings/distinct": [
    "https://github.com/meilisearch/documentation/edit/main/learn/index_settings/distinct.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/index_settings/displayed_searchable_attributes": [
    "https://github.com/meilisearch/documentation/edit/main/learn/index_settings/displayed_searchable_attributes.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/index_settings/synonyms": [
    "https://github.com/meilisearch/documentation/edit/main/learn/index_settings/synonyms.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/index_settings/typo_tolerance": [
    "https://github.com/meilisearch/documentation/edit/main/learn/index_settings/typo_tolerance.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/security/master_api_keys": [
    "https://github.com/meilisearch/documentation/edit/main/learn/security/master_api_keys.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/security/tenant_tokens": [
    "https://github.com/meilisearch/documentation/edit/main/learn/security/tenant_tokens.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/update_and_migration/algolia_migration": [
    "https://www.algolia.com/api-keys",
    "https://github.com/meilisearch/documentation/edit/main/learn/update_and_migration/algolia_migration.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/update_and_migration/previous_docs_version": [
    "https://github.com/meilisearch/documentation/edit/main/learn/update_and_migration/previous_docs_version.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/update_and_migration/updating": [
    "https://github.com/meilisearch/documentation/edit/main/learn/update_and_migration/updating.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/update_and_migration/versioning": [
    "https://github.com/meilisearch/engine-team/blob/main/resources/versioning-policy",
    "https://github.com/meilisearch/documentation/edit/main/learn/update_and_migration/versioning.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/advanced/asynchronous_operations": [
    "https://github.com/meilisearch/documentation/edit/main/learn/advanced/asynchronous_operations.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/advanced/filtering": [
    "https://github.com/meilisearch/documentation/edit/main/learn/advanced/filtering.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/advanced/faceted_search": [
    "https://meilisearch-staging.vercel.app/books.json",
    "https://github.com/meilisearch/documentation/edit/main/learn/advanced/faceted_search.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/advanced/geosearch": [
    "https://github.com/meilisearch/documentation/edit/main/learn/advanced/geosearch.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/advanced/pagination": [
    "https://github.com/meilisearch/documentation/edit/main/learn/advanced/pagination.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/advanced/sorting": [
    "https://github.com/meilisearch/documentation/edit/main/learn/advanced/sorting.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/advanced/working_with_dates": [
    "https://github.com/meilisearch/documentation/edit/main/learn/advanced/working_with_dates.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/data_backup/snapshots_vs_dumps": [
    "https://github.com/meilisearch/documentation/edit/main/learn/data_backup/snapshots_vs_dumps.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/data_backup/dumps": [
    "https://github.com/meilisearch/documentation/edit/main/learn/data_backup/dumps.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/data_backup/snapshots": [
    "https://github.com/meilisearch/documentation/edit/main/learn/data_backup/snapshots.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/inner_workings/concat": [
    "https://github.com/meilisearch/documentation/edit/main/learn/inner_workings/concat.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/inner_workings/datatypes": [
    "https://github.com/meilisearch/documentation/edit/main/learn/inner_workings/datatypes.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/inner_workings/indexing": [
    "https://github.com/meilisearch/documentation/edit/main/learn/inner_workings/indexing.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/inner_workings/known_limitations": [
    "https://github.com/meilisearch/documentation/edit/main/learn/inner_workings/known_limitations.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/inner_workings/prefix": [
    "https://github.com/meilisearch/documentation/edit/main/learn/inner_workings/prefix.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/inner_workings/storage": [
    "https://github.com/meilisearch/documentation/edit/main/learn/inner_workings/storage.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/inner_workings/tokenization": [
    "https://github.com/meilisearch/charabia/blob/main/CONTRIBUTING",
    "https://github.com/meilisearch/documentation/edit/main/learn/inner_workings/tokenization.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/cookbooks/running_production": [
    "http://your-ip-address/",
    "http://example.com/indexes",
    "https://github.com/meilisearch/documentation/edit/main/learn/cookbooks/running_production.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/cookbooks/postman_collection": [
    "https://meilisearch-staging.vercel.app/postman/meilisearch-collection.json",
    "https://github.com/meilisearch/documentation/edit/main/learn/cookbooks/postman_collection.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/cookbooks/docker": [
    "https://github.com/meilisearch/documentation/edit/main/learn/cookbooks/docker.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/cookbooks/search_bar_for_docs": [
    "https://github.com/meilisearch/documentation/edit/main/learn/cookbooks/search_bar_for_docs.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/cookbooks/http2_ssl": [
    "https://github.com/meilisearch/documentation/edit/main/learn/cookbooks/http2_ssl.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/deployment/aws": [
    "https://github.com/meilisearch/documentation/edit/main/learn/deployment/aws.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/deployment/azure": [
    "https://github.com/meilisearch/documentation/edit/main/learn/deployment/azure.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/deployment/digitalocean": [
    "https://github.com/meilisearch/documentation/edit/main/learn/deployment/digitalocean.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/deployment/gcp": [
    "https://github.com/meilisearch/documentation/edit/main/learn/deployment/gcp.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/deployment/koyeb": [
    "https://github.com/meilisearch/documentation/edit/main/learn/deployment/koyeb.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/deployment/qovery": [
    "https://github.com/meilisearch/documentation/edit/main/learn/deployment/qovery.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/deployment/railway": [
    "https://github.com/meilisearch/documentation/edit/main/learn/deployment/railway.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/experimental/metrics": [
    "https://github.com/meilisearch/documentation/edit/main/learn/experimental/metrics.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/contributing/overview": [
    "https://github.com/meilisearch/meilisearch/blob/main/CONTRIBUTING",
    "https://github.com/meilisearch/integration-guides/blob/main/resources/build-integration",
    "https://github.com/meilisearch/documentation/edit/main/learn/contributing/overview.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs/learn/contributing/contributing_docs": [
    "https://www.netlify.com/",
    "https://github.com/meilisearch/documentation/edit/main/learn/contributing/contributing_docs.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ],
  "https://meilisearch-staging.vercel.app/docs//reference/api/settings": [
    "https://github.com/meilisearch/documentation/edit/main/reference/api/settings.mdx",
    "https://www.linkedin.com/company/meilisearch"
  ]
}
```

**Comment by bidoubiwa** [comment]
The linkedin once are not broken, it's just that they don't want robots to pull their content.

**Comment by maryamsulemani97** [comment]
- The `https://github.com/meilisearch/documentation/edit/main/...` links are for editing a page. They don't work as the mdx files don't exist on main
- The LinkedIn link in the footer works for me. Though, it takes me to the sign-in page
- The Netlify link works for me. We will be removing it with #2285 as the new site won't use Netlify
- `mailto:privacy@meilisearch.com"` works for me too

I'll create a PR for the others

**Comment by bidoubiwa** [comment]
Probably the same reason for linkedin as for netlify

**Comment by maryamsulemani97** [comment]
fixed in #2343
